pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -181,7 +181,7 @@ func (pl *InterPodAffinity) PreScore(
 		// Unless the pod being scheduled has affinity terms, we only
 		// need to process pods with affinity in the node.
 		podsToProcess := nodeInfo.PodsWithAffinity
-		if hasAffinityConstraints || hasAntiAffinityConstraints {
+		if hasPreferredAffinityConstraints || hasPreferredAntiAffinityConstraints {
```

Comment:
nit: update the comments to s/affinity/prefferred affinity/g?

Reply:
thanks, nice catch!